src/components/FicheFilm/FicheFilm.jsx

```diff
@@ -1,19 +1,27 @@
 import { useEffect, useState } from 'react';
 import axios from 'axios';
 import './FicheFilm.css';
+import buttonTrailer from './play.png';
+import star1 from './star1.png';
+import star2 from './star2.png';
+import star3 from './star3.png';
+import star4 from './star4.png';
+import star5 from './star5.png';
+import buttonAdd from './add.png';
 
 export default function FicheFilm() {
   const [title, setTitle] = useState([]);
```

Comment:
If your state is an array then it must be on plural. Plz do it from line 13 to 22.